cypress/integration/browse.spec.js

```diff
@@ -13,8 +13,9 @@ describe('Browse', () => {
     beforeEach(() => {
         // Return to home page before each test
         cy.fixture('route.json').then((route) => {
+            console.log(route)
             home = route;
```

Comment:
`home` should probably be renamed to be more indicative of the values it contains, seems to store more than just home info